pkg/virt-handler/vm.go

```diff
@@ -177,8 +233,14 @@ func MapPersistentVolumes(vm *v1.VM, restClient cache.Getter, namespace string)
 				newDisk.Source.Protocol = "iscsi"
 
 				hostPort := strings.Split(pv.Spec.ISCSI.TargetPortal, ":")
+				ipAddrs, err := net.LookupIP(hostPort[0])
```

Comment:
While the workaround is nice, this is the first code wich removes the ability to run virt-handler on the host, like the kubelet. Just to keep it in mind.